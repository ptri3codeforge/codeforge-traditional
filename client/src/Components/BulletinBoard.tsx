import React from 'react'

const BulletinBoard = () => {
    return (
        <div>
            
        </div>
    )
}

export default BulletinBoard
