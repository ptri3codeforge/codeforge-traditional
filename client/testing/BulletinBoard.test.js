import React from 'react'
import renderer from 'react-test-renderer'
import BulletinBoard from '../src/Components/BulletinBoard'

test('Component should match snapshot', () => {
    const component = renderer.create(
        <BulletinBoard></BulletinBoard>
    )
    let tree = component.toJSON();
    console.log(tree)
    expect(tree).toMatchSnapshot()
})

test('BulletinBoard should be type of object', () => {
    const component = renderer.create(
        <BulletinBoard></BulletinBoard>
    )
    let tree = component.toJSON();
    expect(tree).toMatchSnapshot()
    expect(typeof tree).toBe('object')
})