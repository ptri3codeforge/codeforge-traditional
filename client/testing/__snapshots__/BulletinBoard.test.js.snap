// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`BulletinBoard should be type of object 1`] = `<div />`;

exports[`Component should match snapshot 1`] = `<div />`;
